temporalio/worker/_worker.py

```diff
@@ -313,6 +322,49 @@ def __init__(
                 disable_safe_eviction=disable_safe_workflow_eviction,
             )
 
+        workflow_slot_supplier: temporalio.bridge.worker.SlotSupplier
+        activity_slot_supplier: temporalio.bridge.worker.SlotSupplier
+        local_activity_slot_supplier: temporalio.bridge.worker.SlotSupplier
+
+        if tuner is not None:
```

Comment:
Another approach is to just have `WorkerTuner` have a private `_create_bridge_tuner` method, so something like:

```python
if tuner:
    if max_concurrent_workflow_tasks or max_concurrent_activities or max_concurrent_local_activities:
        raise ValueError(
            "Cannot specify max_concurrent_workflow_tasks, max_concurrent_activities, "
            "or max_concurrent_local_activities when also specifying tuner"
        )
else:
    tuner = WorkerTuner.create_fixed(
        max_concurrent_workflow_tasks=max_concurrent_workflow_tasks,
        max_concurrent_activities=max_concurrent_activities,
        max_concurrent_local_activities=max_concurrent_local_activities,
    )
bridge_tuner = tuner._create_bridge_tuner()
```

That at least moves the defaults and all tuning code to one place